app/src/main/res/layout/partial_banks_list.xml

```diff
@@ -10,36 +9,38 @@
     android:padding="16dp">
 
     <TextView
-        android:id="@+id/tvName2"
+        android:id="@+id/bankName_tv"
+        android:layout_toStartOf="@id/favouritesIcon"
+        android:layout_alignParentStart="true"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:text="@string/bank_name"
         android:textColor="@android:color/white"
-        android:textSize="20sp" />
+        android:textSize="16sp" />
 
     <TextView
-        android:id="@+id/tvAmount"
+        android:id="@+id/bankAddress_tv"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_below="@id/tvName2"
+        android:layout_below="@id/bankName_tv"
+        android:layout_marginTop="5dp"
         android:text="@string/bank_address"
         android:textColor="@android:color/white"
-        android:textSize="20sp" />
-
+        android:textSize="16sp" />
 
 
     <ImageView
-        android:id="@+id/ivMinus2"
+        android:id="@+id/favouritesIcon"
```

Comment:
The naming conventions used across the xml files isn't right. Given the fact that using kotlin extensions these variable names are available as bindings you should use camel case for all ids

Reply:
Sure, I'll take care of that.